libfreeipmi/include/freeipmi/spec/ipmi-lan-configuration-parameters-spec.h

```diff
@@ -50,13 +50,44 @@ extern "C" {
 #define IPMI_LAN_CONFIGURATION_PARAMETER_RMCPPLUS_MESSAGING_CIPHER_SUITE_PRIVILEGE_LEVELS 24
 #define IPMI_LAN_CONFIGURATION_PARAMETER_DESTINATION_ADDRESS_VLAN_TAGS                    25
 #define IPMI_LAN_CONFIGURATION_PARAMETER_BAD_PASSWORD_THRESHOLD                           26
+#define IPMI_LAN_CONFIGURATION_PARAMETER_IPV6_SUPPORT                                     50
+#define IPMI_LAN_CONFIGURATION_PARAMETER_IPV6_ADDRESSING_ENABLE                           51
```

Comment:
Oh and I think the second one is "enables" with a plural 's', sorry, you're gonna find I'm a stickler for sticking to the specification :-|